src/settings.js

```diff
@@ -227,7 +290,111 @@ const settings = {
             await match.accessible.Button('Closed Loop').tap();
             await expect(match.accessible.Button('Closed Loop')).toHaveValue('0');
         }
-        await match.accessible.ButtonBarButton('Done').tap();
+        if (startedFromHomeScreen){
+            await returnToHomeScreen();
+        }
+    },
+    /**
+     * @summary Defaults that can be used to apply to all settings
+     * @example await settings.Apply(settings.Defaults)
+     */
+    Defaults: {
+        /**
+         * @summary DeliveryLimits: { maxBolus: '10.0', maxBasalRate: '3.0' }
+         */
+        DeliveryLimits: { maxBolus: '10.0', maxBasalRate: '3.0' },
+        /**
+         * @summary BasalRates: [{ time: '12:00 AM', unitsPerHour: '0.1' }]
+         */
+        BasalRates: [{ time: '12:00 AM', unitsPerHour: '0.1' }],
+        /**
+         * @summary SuspendThreshold: { threshold: '75' }
+         */
+        SuspendThreshold: { threshold: '75' },
+        /**
+         * @summary SelectInsulinModel: { value: 2, name: 'Rapid-Acting – Children' }
+         */
+        SelectInsulinModel: { value: 2, name: 'Rapid-Acting – Children' },
+        /**
+         * @summary CarbRatios: [{ time: '12:00 AM', carbGramsPerInsulinUnit: '8' }]
+         */
+        CarbRatios: [{ time: '12:00 AM', carbGramsPerInsulinUnit: '8' }],
+        /**
+         * @summary InsulinSensitivities: [{ time: '12:00 AM', bgValuePerInsulinUnit: '500' }]
+         */
+        InsulinSensitivities: [{ time: '12:00 AM', bgValuePerInsulinUnit: '500' }],
```

Comment:
[nit] Consider making the defaults "reasonable" values. Many of these are already, but `InsulinSensitivities`, `CorrectionRanges`, and `PreMealCorrectionRange` seem unlikely.

Reply:
I agree - most of this is still a WIP as it comes down to actually being able to select the more reasonable values. This will happen when we have accessible pickers